pkg/workflows/wasm/pb/wasm.proto

```diff
@@ -24,6 +24,7 @@ message Request {
     ComputeRequest computeRequest = 3;
     google.protobuf.Empty specRequest = 4;
   }
+  string triggerId = 5;
```

Comment:
Why do we need this? This proto is not used by NoDAG, is it...?

Reply:
To know which trigger fired. I'll add a TODO so we can support multiple of the same trigger. Currently, I'm using the id of the trigger to know which callback to make.

Reply:
I'll revert, I allowed you to wrap V2 calls as V1 calls, but we're not going that route. I forgot to revert the file.